waspc/data/Generator/templates/server/src/core/auth/prismaMiddleware.js

```diff
@@ -2,7 +2,9 @@
 import { hashPassword } from '../auth.js'
 import AuthError from '../AuthError.js'
 
+{=# isUsernameAndPasswordAuthEnabled  =}
 const USERNAME_FIELD = 'username'
+{=/ isUsernameAndPasswordAuthEnabled  =}
```

Comment:
We could have an error if username is on the User entity, but usernameandPassword auth is _not_ enabled (due to use below). Maybe we just don't wrap this?

Reply:
Updated to use `isUsernameOnUserEntity` to keep it consistent.